testsuite/src/main/java/io/netty/testsuite/transport/socket/SocketShutdownOutputByPeerTest.java

```diff
@@ -68,6 +70,9 @@ public void testShutdownOutput(ServerBootstrap sb) throws Throwable {
             Thread.sleep(100);
             assertEquals(1, h.halfClosureCount.intValue());
         } finally {
+            if (s != null) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Change this condition so that it does not always evaluate to "true" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS2583)